tests/e2e/Makefile

```
@@ -12,34 +13,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-GO_CMD ?= go
+.PHONY: test-e2e ginkgo
 
-include $(CURDIR)/versions.mk
+GINKGO_ARGS ?=
+LOG_ARTIFACTS_DIR ?= $(CURDIR)/e2e_logs
 
-E2E_RUNTIME ?= docker
+ginkgo:
```

Should this be:
```suggestion
ginkgo: $(CURDIR)/bin/ginkgo
```
so that we don't ALWAYS install ginkgo?